sktime/forecasting/prophet.py

```diff
@@ -0,0 +1,394 @@
+        self.holidays = holidays
+        self.seasonality_mode = seasonality_mode
+        self.seasonality_prior_scale = float(seasonality_prior_scale)
+        self.changepoint_prior_scale = float(changepoint_prior_scale)
+        self.holidays_prior_scale = float(holidays_prior_scale)
+        self.mcmc_samples = mcmc_samples
+        self.interval_width = interval_width
+        self.uncertainty_samples = uncertainty_samples
+        self.stan_backend = stan_backend
+
+        super(Prophet, self).__init__()
+
+        # import inside method to avoid hard dependency
+        from fbprophet.forecaster import Prophet as _Prophet
+
+        self._forecaster = _Prophet(
+            growth=growth,
+            changepoints=changepoints,
+            n_changepoints=n_changepoints,
+            changepoint_range=changepoint_range,
+            yearly_seasonality=yearly_seasonality,
+            weekly_seasonality=weekly_seasonality,
+            daily_seasonality=daily_seasonality,
+            holidays=holidays,
+            seasonality_mode=seasonality_mode,
+            seasonality_prior_scale=seasonality_prior_scale,
+            holidays_prior_scale=holidays_prior_scale,
+            changepoint_prior_scale=changepoint_prior_scale,
+            mcmc_samples=mcmc_samples,
+            interval_width=interval_width,
+            uncertainty_samples=uncertainty_samples,
+            stan_backend=stan_backend,
+        )
+
+    def fit(self, y, X=None, fh=None, **fit_params):
+        """Fit to training data.
+        Parameters
+        ----------
+        y : pd.Series
+            Target time series to which to fit the forecaster.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables.
+        fh : int, list or np.array, optional (default=None)
+            The forecasters horizon with the steps ahead to to predict.
+        Returns
+        -------
+        self : returns an instance of self.
+        """
+        self._check_changepoints()
+        y, X = check_y_X(y, X, enforce_index_type=pd.DatetimeIndex)
+        self._set_y_X(y, X)
+        self._set_fh(fh)
+
+        # We have to bring the data into the required format for fbprophet:
+        df = pd.DataFrame(y.rename("y"))
+        df["ds"] = y.index
+
+        # Add seasonality
+        if self.add_seasonality:
+            self._forecaster.add_seasonality(**self.add_seasonality)
+
+        # Add country holidays
+        if self.add_country_holidays:
+            self._forecaster.add_country_holidays(**self.add_country_holidays)
+
+        # Add regressor (multivariate)
+        if X is not None:
+            df = df.merge(X, left_index=True, right_on=X.index)
+            for col in X.columns:
+                self._forecaster.add_regressor(col)
+
+        self._forecaster.fit(df=df, **fit_params)
+        self._is_fitted = True
+        return self
+
+    def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
```

Comment:
todo: @aiwalter 
if alpha != None, raise exception. because alpha can only be provided when fitting.